source/robot_model/src/Model.cpp

```diff
@@ -383,11 +377,21 @@ Model::inverse_velocity(const std::vector<state_representation::CartesianTwist>&
   this->hessian_.setFromTriplets(coefficients.begin(), coefficients.end());
   //set the gradient
   this->gradient_.head(nb_joints) = -parameters.proportional_gain * delta_r.transpose() * jacobian;
-  // update qp_constraints
-  this->lower_bound_constraints_(4 * nb_joints) = parameters.epsilon;
+  // update minimal time as dt expressed in seconds
+  this->lower_bound_constraints_(3 * nb_joints) = parameters.dt.count() / 1e9;
```

Comment:
It would be nice if you could get this 1e9 scalar from the chrono type - I think `parameters.dt.period.den` might work

Reply:
there don't seem to be any period attribute. I will try to find the correct stuff

Reply:
Ah ok, I hadn't tested it but just suspected `duration` type has a `period` attribute which is a `std::ratio`, might not be publicly visible though.

In that case the preferred way to do this line would definitely be:
std::chrono::duration_cast<std::chrono::seconds>(parameters.dt).count()

Reply:
Yes this would be a nice solution. I am doing a similar stuff in `state_representation` in the conversion functions. I should make a PR to do the same there.

Reply:
ok this was a bit more complex but `duration_cast<duration<float>>(parameters.dt).count();` does the trick